inventory.lua

```diff
@@ -2,6 +2,30 @@ local S = digilines.S
 
 local pipeworks_enabled = minetest.get_modpath("pipeworks") ~= nil
 
+-- Signals which will be sent in a single batch
+local batched_signals = {}
+-- Maximum interval from the previous signal to include the current one into batch (in seconds)
+local interval_to_batch = 0.1
+-- Maximum number of signals in batch
+local max_signals_in_batch = 100
+
+-- Sends the current batch message of a Digiline chest
+-- pos: the position of the Digilines chest node
+-- channel: the channel to which the message will be sent
+local function send_and_clear_batch(pos, channel)
+	local pos_text = vector.to_string(pos)
```

Comment:
`minetest.hash_node_position(pos)` might be a better choice. It returns an integer for faster table lookups.